testdata/v2_atlantis_depends_on/terraform/envs/test/vpc/main.tf

```diff
@@ -15,8 +15,15 @@ module "vpc" {
 
 module "my_module" {
   source = "../../../modules/my_module"
+  depends_on = [
+    module.vpc,
+  ]
```

Comment:
can you also `make install` and then in `lift-and-shift` modify it and run `~/go/bin/fogg apply -upgrade && make pre-commit` and go to the `envs/../...` run `terraform init` / `terraform plan` in it to confirm syntax is all good?